src/frontend/utils/telemetry/FrontendTracer.ts

```diff
@@ -6,11 +6,10 @@ import { WebTracerProvider } from '@opentelemetry/sdk-trace-web';
 import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base';
 import { registerInstrumentations } from '@opentelemetry/instrumentation';
 import { getWebAutoInstrumentations } from '@opentelemetry/auto-instrumentations-web';
-import { Resource, browserDetector } from '@opentelemetry/resources';
+import { Resource, browserDetector, detectResourcesSync } from '@opentelemetry/resources';
 import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
 import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
 import { SessionIdProcessor } from './SessionIdProcessor';
-import { detectResourcesSync } from '@opentelemetry/resources/build/src/detect-resources';
```

Comment:
It looks like there could be a reason this import was used instead of the current diff?

Reply:
Both reference the exact same function, from experience I'd guess this was a VSCode auto-import error.